core/src/OAuth2/MsalTokenResponse.cs

```diff
@@ -68,11 +71,28 @@ internal class MsalTokenResponse : OAuth2ResponseBase
         public string IdToken { get; set; }
 
         [DataMember(Name = TokenResponseClaim.ExpiresIn, IsRequired = false)]
-        public long ExpiresIn { get; set; }
+        public long ExpiresIn
```

Comment:
This can be unit tested as well: - the logic itself and maybe it's a good time to start having serialization tests (e.g. to from json with / without ExpiresIn)

Reply:
Agreed.  Added [569774] (assigned to me) to get serialization classes factored properly and add unit tests around serialization/deserialization/parsing.  Further modifications here should probably want until Petro's other cache PR is merged.